proto-java-plugin/src/main/java/io/github/lwlee2608/proto/plugin/DownloadMojo.java

```diff
@@ -26,54 +37,70 @@ public class DownloadMojo extends AbstractMojo {
     @Parameter(
             name = "protocVersion",
             property = "protoc.version",
-            defaultValue = "3.4.0"
+            defaultValue = "3.22.3"
     )
     private String protocVersion;
 
     @Parameter(
             name = "os",
             property = "os.identifier",
```

Comment:
I think we should just remove `os.identifier` since we can auto-detect it now.

Reply:
there're few os architecture which I don't have environments to test. 
When auto-detect doesn't work properly, user has the option to specify the OS.

Reply:
@hsrky One of the design principle of this project is to reduce as much configuration as possible. We can put a disclaimer on what OS we support. There's only a handful of OS majority of the developer use, and I believe we can have sufficient coverage for most of them.

 Maybe you can refer to this [plugin](https://github.com/trustin/os-maven-plugin)